examples/seg/deeplabv3/bulid_dataset/bulid_seg_data.py

```diff
@@ -0,0 +1,58 @@
+import argparse
```

Comment:
files in `bulid_dataset`  are all about preprocess, right? need a better name.
btw, "build", not "bulid".

Reply:
`bulid_dataset ` rename to `preprocess`